src/Utility/Xml.php

```diff
@@ -142,7 +142,7 @@ protected static function _loadXml($input, $options)
     {
         $hasDisable = function_exists('libxml_disable_entity_loader');
         $internalErrors = libxml_use_internal_errors(true);
-        if ($hasDisable && !$options['loadEntities']) {
+        if ($hasDisable && empty($options['loadEntities'])) {
```

Comment:
>  use of empty($options['loadEntities']) instead of !$options['loadEntities']

Why though? What does this change?

Reply:
If not exists $options['loadEntities'] Notice is raised:
```
2018-06-02 02:48:02 Notice: Notice (8): Undefined index: loadEntities in [C:\usr\0xFF\bambu\app\vendor\cakephp\cakephp\src\Utility\Xml.php, line 184]
```

Reply:
How would that happen? The public methods set a default value.

Reply:
Ok, I see.
I based loadHtml in loadXml thats is calling by build, so in loadHtml $options don't have default value.